manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -69,33 +81,144 @@ public void init(Container container) throws Exception {
+        if (!Files.isRegularFile(mapSettingsPath)) {
+            LOG.warning("Map settings file not found '" + mapSettingsPath.toAbsolutePath() + "', falling back to built in map settings");
+            mapSettingsPath = persistenceService.getStorageDir().resolve("mapsettings.json");
+            if(!Files.isRegularFile(mapSettingsPath)){
+                mapSettingsPath = Paths.get(OR_MAP_SETTINGS_PATH_DEFAULT).resolve("mapsettings.json");
+                if(!Files.isRegularFile(mapSettingsPath)){
+                    LOG.severe("Map settings file not found, map functionality will not work");
+                }
+            }
+        }
+
+        // Will throw if failed, stopping startup fast and hard
+        try {
+            loadMapSettingsJson();
+        }catch (Exception e){
+            LOG.severe("Could not load MapSettings.json file. Map functionality will be limited. Error: "+ e.getMessage());
+        }
+
+        try {
+            loadManagerConfigJson();
+        }catch (Exception e){
+            LOG.severe("Could not load manager_config.json file. Appearance will be default. Error: "+ e.getMessage());
+        }
     }
 
     @Override
     public void start(Container container) throws Exception {
-        /* code not overridden yet */
+
+        // Check if the configuration contains references to files that are located in the deployment, and if they are,
+        // move them to the storageDir.
+        ObjectNode changedConfig = this.checkAndFixImageReferences(this.managerConfig);
+
+        if(changedConfig != null){
+            saveManagerConfigFile(changedConfig);
+        }
+
     }
 
     @Override
     public void stop(Container container) throws Exception {
         /* code not overridden yet */
     }
 
+    protected void loadMapSettingsJson() throws Exception {
+        try {
+            this.mapConfig = (ObjectNode) ValueUtil.JSON.readTree(Files.readAllBytes(mapSettingsPath));
+        } catch (Exception ex) {
+            LOG.log(Level.SEVERE, "Failed to extract map config from: " + mapSettingsPath.toAbsolutePath(), ex);
+            throw ex;
+        }
+    }
+
+    public Optional<File> getManagerConfigFile(){
+        File file = getManagerConfigPath().toFile();
+
+        if(file.exists() && !file.isDirectory()){
+            return Optional.of(file);
+        }else{
+            file = pathPublicRoot
+                    .resolve("manager_config.json")
+                    .toFile();
+            if (file.exists() && !file.isDirectory()){
+                return Optional.of(file);
+            }
+        }
+        return Optional.empty();
+    }
+
+    public ObjectNode getManagerConfig(){
+        return this.managerConfig;
+    }
+
+    public Path getManagerConfigPath(){
+        return persistenceService.getStorageDir()
+                .resolve("manager")
+                .resolve("manager_config.json");
+    }
+
+
+    protected void loadManagerConfigJson() throws IOException {
```

Comment:
It is perfectly valid to not have a manager config so exceptions shouldn't be thrown or severe log messages, info is appropriate.

Reply:
Fixed, logging set to `INFO`